client/my-sites/email/email-providers-comparison/in-depth/data.tsx

```diff
@@ -7,33 +7,34 @@ import { getGoogleMailServiceFamily } from 'calypso/lib/gsuite';
 import { getTitanProductName } from 'calypso/lib/titan';
 import type { EmailProviderFeatures } from 'calypso/my-sites/email/email-providers-comparison/in-depth/types';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-function
-const noop = () => {};
-
 export const professionalEmailFeatures: EmailProviderFeatures = {
 	slug: 'professional-email',
 	name: getTitanProductName(),
+	description: translate( 'Integrated email solution for your WordPress.com site' ),
 	logo: <Gridicon className="professional-email-logo" icon="my-sites" />,
-	tools: translate( 'Integrated email management, Inbox, calendar and contacts' ),
-	storage: translate( '30GB storage' ),
-	importing: translate( 'One-click import of existing emails and contacts' ),
-	support: translate( '24/7 support via email' ),
-	selectCallback: noop,
+	list: {
+		importing: translate( 'One-click import of existing emails and contacts' ),
+		storage: translate( '30GB storage' ),
+		support: translate( '24/7 support via email' ),
+		tools: translate( 'Integrated email management, Inbox, Calendar and Contacts' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Integrated email management, Inbox, calendar and contacts' )` *ES Score: 13*